modules/openapi-generator/src/main/resources/Java/libraries/webclient/api.mustache

```diff
@@ -53,8 +53,8 @@ public class {{classname}} {
         this.apiClient = apiClient;
     }
 
-    {{#operation}}{{#useSingleRequestParameter}}{{#hasParams}}{{^hasSingleParam}}
-    public class {{#lambda.titlecase}}{{operationId}}{{/lambda.titlecase}}Request {
+    {{#operation}}{{#singleRequestParameter}}{{#hasParams}}{{^hasSingleParam}}
+        public {{#staticRequest}}static {{/staticRequest}}class {{#lambda.titlecase}}{{operationId}}{{/lambda.titlecase}}Request {
```

Comment:
Please remove the whitespace that has been added at the beginning of this line.

Reply:
Removed on [1f30b5f](https://github.com/OpenAPITools/openapi-generator/pull/20590/commits/1f30b5fb34bb9bccf34a875f34fd46e35dc4191e)